### PR TITLE
Fix fontfamily classes

### LINKED ISSUE #379: Remove outdated and unused fonts
**Desired Behavior**

- remove woff to reduce footprint
- Remove book font as it is unused

### PR DESCRIPTION
**Pull request by fthobe**
serif and serif-md where loading wrong fallback system font classes

## Summary

Hey, on testing I noticed I missed the fall back font classes. This is the final final PR for the font issues. 

  Fixes #379 

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [X] I have written a thorough PR description.
- [X] I have kept my commits small and atomic.
- [X] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

**Review by jarednorman — APPROVED** [review]
Good catch.

**Comment by fthobe** [comment]
@tvdeyen do the merge Magic

**Comment by jarednorman** [comment]
We normally wait for two reviews. Please don't ping maintainers unnecessarily.

**Review by tvdeyen — APPROVED** [review]

**Pull request merged**
